assets/scss/app.scss

```diff
@@ -0,0 +1,11 @@
+@tailwind base;
```

Comment:
Let's comment this with notes on how to remove if they don't want to use tailwind